rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -186,13 +187,17 @@ def _decode_withdrawal(
         user = hex_or_bytes_to_address(tx_log.topics[2])
         to = hex_or_bytes_to_address(tx_log.topics[3])
         return_event, withdraw_event = None, None
-        if not self.base.any_tracked([user, to]):
+        if (
+            not (is_weth_user := user in self.eth_gateways) and
```

Comment:
Is it called weth user in the contract? I would like to avoid any complexity that makes people mistake how this is supposed to work. Afaics it's always native token right?

Also is it called eth_gateways or shouldn't it be `native_gateways`?

```suggestion
            not (is_wnative_user := user in self.eth_gateways) and
```